tests/test-cells/src/widget.spec.ts

```diff
@@ -357,10 +357,10 @@ describe('cells/widget', () => {
       it('should not throw an error (full test in input area)', () => {
         const widget = new Cell({ model, contentFactory }).initializeState();
         expect(() => {
-          widget.setPrompt(void 0);
+          widget.setPrompt(void 0 as any);
```

Comment:
Should we change the API to still accept `undefined` and `null`? If not, do we still want to keep the tests here?

Reply:
I'd say remove the test and not accept `undefined` or `null`.